google/cloud/forseti/services/dao.py

```diff
@@ -262,6 +262,19 @@ def define_model(model_name, dbengine, model_seed):
                primary_key=True),
```

Comment:
@blueandgold I'm still somewhat unfamiliar with the model stuff and want to confirm that Im not missing anything I need here.  For instance I probably need bindings or queries?  Lets chat tomorrow